dll/win32/shell32/wine/pidl.c

```diff
@@ -1722,6 +1722,62 @@ LPITEMIDLIST _ILCreateGuidFromStrA(LPCSTR szGUID)
 }
 #endif
 
+#ifdef __REACTOS__
+static const struct {
+    LPCWSTR pszName;
+    LPCGUID guid;
+} ShellShortcuts[] =
```

Comment:
Any reason why you don't use `CSIDL_Data` from `dll/win32/shell32/wine/shellpath.c` ?

Reply:
Thanks for the tip, I'll upload the changes soon.